torchvision/ops/boxes.py

```diff
@@ -311,6 +311,50 @@ def generalized_box_iou(boxes1: Tensor, boxes2: Tensor) -> Tensor:
     return iou - (areai - union) / areai
 
 
+# Implementation inspired from the generalized_box_iou one.
+# TODO: Some refactoring and homogenization could be done with
+# the loss function in diou_loss.
```

Comment:
According to https://github.com/pytorch/vision/pull/5776#issuecomment-1102613291 we should do that here, correct?

Reply:
Yes, will be removed. :ok_hand: